ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -233,8 +241,57 @@ public void storeAccountStats(StatsWrapper statsWrapper) throws SQLException {
       }
     }
     batch.flush();
-    storeMetrics.publishTimeMs.update(System.currentTimeMillis() - startTimeMs);
     storeMetrics.batchSize.update(batchSize);
+    storeMetrics.insertAccountStatsTimeMs.update(System.currentTimeMillis() - startTimeMs);
```

Comment:
nit: either now or later, this method needs to be divided into smaller methods. This method is way too long, and seems to be doing multiple things, making it an ideal candidate for refactoring.